LICENSE.md

```diff
@@ -54,3 +54,6 @@ The Python modules used by Mbed tools are used under the following licenses:
 - [pywin32](https://pypi.org/project/pywin32/) - PSF
 - [wmi](https://pypi.org/project/WMI/) - MIT
 - [psutil](https://pypi.org/project/psutil/) - BSD
+- [click](https://pypi.org/project/click/) - BSD-3-Clause
+- [cryptography](https://pypi.org/project/cryptography/) - BSD, Apache-2.0
+- [jwcrypto](https://pypi.org/project/jwcrypto/) - LGPLv3+
```

Comment:
@ARMmbed/mbed-os-tools is this acceptable ?

Reply:
I cannot tell whether this is acceptable from a legal point of view (IANAL), but it looks like this follows the same pattern as for other tools in this file. Has anyone done a TPIP check on any of this in the past? I would advise that the legal dept. reviews this before it goes out.

Reply:
@romanjoe we should use different package or check if this is not dual licensed? We won't accept LGPLv3+.

Reply:
@mark-edgeworth, @0xc0170  can double check with the legal and provide a feedback?

Reply:
Our license guide https://os.mbed.com/docs/mbed-os/v5.13/contributing/license.html specifies the code needs to under permissive license. Same should be applied to python dependencies

Reply:
I'll double check today

Reply:
> Our license guide https://os.mbed.com/docs/mbed-os/v5.13/contributing/license.html specifies the code needs to under permissive license. Same should be applied to python dependencies

Checked, can't accept this module with its license.

Reply:
@0xc0170  we are rewriting the PR to remove the provisioning scripts that rely on jwcrypto from it, that way we will be able to resubmit without dependency on jwcrypto. Expect a new PR soon.
 thanks,
   Christian,